### DISCUSSION TITLE
【求助】突然穿越，怎么回去！！！

### DISCUSSION THREAD

**Discussion by shengwang999** [share121/inter-knot#2961] [其他]
今天早上买早餐的时候突然感觉头脑发晕，再睁眼就到这里了，这里是哪里？？怎么回去？急急急！！！
![Yuan Shen 原神 Screenshot 2024 08 19 - 04 08 42 76](https://github.com/user-attachments/assets/cbca1f80-d034-4ecf-97ba-73e035955645)

**Comment by chenx198** [comment]
对不起，您的委托不在新爱丽都的业务范围内，请拨打往生堂热线客服114514进行咨询。

**Comment by Legendary-agent** [comment]
【灌水】看起来山清水秀的，没有空洞的地方真好啊。

**Comment by fuqiam** [comment]
这个委托我接了

**Comment by lixiao189** [comment]
你去河里钓一下鱼，看看会不会钓上来一个极度贪吃的白精灵

**Comment by Ziew-Ryan** [comment]
对不起，您不在新艾利都政府管辖范围内，具体事宜请致电枫丹沫芒宫服务热线
Sorry, you are not under the jurisdiction of the New Eridu government, please call the Fontaine Momang service hotline for more information

**Comment by RaphaelHider** [comment]
我查了一下资料，这可能是空洞灾害之前的世界，神奇的是你还能上绳网，建议直接联系治安局，这种时候盗洞客的身份暴露也无关紧要了，跟官方合作才能有更大几率生还。

**Comment by yivitte** [comment]
奇怪，你掉到的这个地方看起来怎么好像我之前玩过的一个二字游戏？
